tools/integration_tests/util/creds_tests/testdata/revoke_permission_and_creds.sh

```diff
@@ -12,9 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Delete service account after testing
+# Delete key file after testing
 SERVICE_ACCOUNT=$1
 KEY_FILE=$2
+
 gcloud auth revoke $SERVICE_ACCOUNT
-gcloud iam service-accounts delete $SERVICE_ACCOUNT
+# Crete key file output
+# e.g. created key [key_id] of type [json] as [key_file_path] for [service_account]
+# capturing third word from the file to get key-id
+# e.g. capture [KEY_ID]
+KEY_ID=$(cat ~/key_id.txt | cut -d " " -f 3)
```

Comment:
I'm not aware of this code, so naive question.. Is `~/key_id.txt` supposed to be replaced by `$KEY_FILE` here ?

Reply:
From the code, it looks like I'm wrong. Never mind.